codechain/config/presets/config.prod.toml

```diff
@@ -1,9 +1,9 @@
 [codechain]
 quiet = false
-db_path = "db"
-keys_path = "keys"
+base_path = "."
 chain = "mainnet"
 
+
```

Comment:
You don't need this change.

Reply:
Without this change, the code in the line https://github.com/CodeChain-io/codechain/pull/1499/files/3c283f526dfb43ca1261fa5a77f090e2626b41c7#diff-22a0b3dcef5d27fab89655ebdb330967R257 could be a cause of panic.
Do I need to remove this change and use `unwrap_or` for the line mentioned above?

Reply:
Why does it panic when you remove an empty line?

Reply:
Oh I'm sorry I misunderstood that you mentioned about the `+base_path = "."` change.